package/libnetwork/src/libnetwork.c

```diff
@@ -199,6 +267,11 @@ static int network_progress_check(network_progress_t *progress, curl_off_t bytes
     progress->count = 0;
   }
 
+  time_t now = time(NULL);
+  if (now - last_xfer_time >= progress->gga_xfer_secs) {
+    curl_easy_pause(progress->curl, CURLPAUSE_CONT);
```

Comment:
This is potentially a problem - calling CONT on a handle that's not paused? Thinking of the case where interval is 0 and there is no pausing.

Reply:
guard it with `gga_xfer_secs != 0`?